tests/taipy/gui/gui_specific/test_expression.py

```diff
@@ -29,38 +31,57 @@ def test_expression_text_control_int(gui: Gui, helpers):
 
 
 def test_expression_text_control_1(gui: Gui, helpers):
+    gui._set_frame(inspect.currentframe())
     gui._bind_var_val("x", 10)
     gui._bind_var_val("y", 20)
     md_string = "<|{x + y}|>"
-    expected_list = ["<Field", 'dataType="int"', 'defaultValue="30"', "value={tp_x_y_0}"]
+    expected_list = [
+        "<Field",
+        'dataType="int"',
+        'defaultValue="30"',
+        "value={tp_x_y_TPMDL_tests_DOT_taipy_DOT_gui_DOT_gui_specific_DOT_test_expression_0}",
```

Comment:
Waow !!! That's quite verbose ....

Reply:
very very verbose since this is the full module name :D

Reply:
Your assumption is quite right Fred.

Right now, on the variable side, if the variable is from the default frame, no module suffix will be added. But in other module, the suffix will be added. On the evaluation, I also added a suffix to the expression to differentiate it. But currently there is no exclusion for the default frame on the expression encoding, hence the long long name in this case.

I really like your suggestion on a directory for module_name and shorten index. I will implement it if @FabienLelaquais also like the idea.